command/crypto/jose/jose.go

```diff
@@ -64,7 +64,7 @@ $ step crypto jwt sign --key p256.priv.json --iss "joe" --aud "bob" \
 	}
 }
 
-func formatAction(ctx *cli.Context) error {
+func formatAction(_ *cli.Context) error {
```

Comment:
We can get rid of it or:
```suggestion
func formatAction(*cli.Context) error {
```